docs/framework/data/wcf/index.md

```diff
@@ -77,17 +77,17 @@ What can I do with it?
 
 - [Overview](wcf-data-services-overview.md)
 
-- [Whitepaper: Introducing OData](https://go.microsoft.com/fwlink/?LinkId=220867)
+- [Whitepaper: Introducing OData](../../wcf/index.md)
 
 - [Application Scenarios](application-scenarios-wcf-data-services.md)
 
 I want to use Silverlight...
```

Comment:
Remove this entire section about silverlight

Reply:
I'm removing it